Dockerfile.old-test-suite

```diff
@@ -1,27 +1,11 @@
-FROM openbuildservice/base
-
-# FIXME: https://bugzilla.opensuse.org/show_bug.cgi?id=957818
-RUN rm -rf /var/cache/zypp/*
+FROM openbuildservice/frontend-base
```

Comment:
If you base this image on frontend-base it will have to be constantly rebuild (like the frontend image) whenever there are changes in the bundle. Not sure this is what you have intended.